WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionSheetViewController.swift

```diff
@@ -69,9 +69,17 @@ class ReaderSortingActionSheetOptionControl: ClosureControl {
         isSelected = option.checked
         imageView.image = option.image
         label.text = option.title
+
+        isAccessibilityElement = true
+        checkView.isAccessibilityElement = false
+        imageView.isAccessibilityElement = false
+        label.isAccessibilityElement = false
+
         accessibilityIdentifier = option.identifier
-        accessibilityLabel = option.identifier
+        accessibilityLabel = option.sortingOption.accessibilityLabel
         accessibilityHint = option.sortingOption.accessibilityHint
+        accessibilityValue = option.sortingOption.accessibilityValue
+        accessibilityTraits = UIAccessibilityTraits.button
```

Comment:
I am thinking maybe to include `option.checked` for setting the accessibility properties, to tell the user that one of the items is selected/checked. Would it be better to have `accessibilityValue` something like "selected" or should I put it directly in `accessibilityLabel`?